libraries/helpers.rb

```diff
@@ -1,7 +1,6 @@
 #
 # Cookbook:: postgresql
 # Library:: helpers
-# Author:: David Crane (<davidc@donorschoose.org>)
```

Comment:
I've removed David as the Author here, as none of this code is actually his anymore. We've refactored so much of it. That the methods that he wrote no longer exist